src/domains/printable.ml

```diff
@@ -470,7 +470,7 @@ module type ChainParams = sig
   val names: int -> string
 end
 
-module Chain (P: ChainParams): S with type t = int =
```

Comment:
Why is this signature removal required?

Reply:
Oops, it's not :) I had added signatures all over the place while tinkering, then removed them again, and I guess also accidentally removed this.